src/odemis/driver/tmcm.py

```diff
@@ -427,22 +419,39 @@ def __init__(self, name, role, port, axes, ustepsize, address=None,
             self._updateTemperatureVA() # make sure the temperature is correct
             self._temp_timer.start()
 
-    # def _is_already_ref(self, an):
-    #     if self._abs_encoder[an] is True:
-    #         return True
-    #     # Check the user variable is in the "right format"
-    #     self.aref
-    #     # if not -> reset UVXXX to 0, and return False
-    #     gparam = 117
-    #     self.SetGlobalParam(2, gparam, 0)
-    #     return F
-    #     # return UVXXX value at [an]
-    #     an = self.aref
-    #     return an
-
-    def _set_ref(self, an, value):
-        # read the curren value of UVXXX
-        #update UVXXX with an = value
+    def _is_already_ref(self, an):
+        """
+
+        Args:
+            an: axis number
+
+        Returns (boolean): if the axis is referenced or not
+
+        """
+        if self._abs_encoder[an] is True:
+            return True
+        arefd = self.GetGlobalParam(2, 117)
```

Comment:
117 seems like a magic number. I'd suggest you use a constant, maybe like `AREF_USER_VAR`, and next to it, you explain why you choose that number.